.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,20 @@
-### What changes were proposed in this pull request?
+### What is the goal of this PR?
+
+
+### Links (JIRA, Notion, Miro)
```

Comment:
Would put this either in a smaller heading or at the bottom given that it doesn't need to be the most important thing for the reviewer to look at before starting the review.